src/containers/Home.jsx

```diff
@@ -101,5 +118,26 @@ const mapQueriesToProps = () => ({
   }
 })
 
+const mapMutationsToProps = () => ({
+  // createCampaign: (campaign) => ({
```

Comment:
Campaign admin is reverted - just forgot to delete the campaign mutation after copy pasting this part here.